public/page2/dom.js

```diff
@@ -1,80 +1,82 @@
-const searchBtn =document.querySelector(".search-button")
-const searchInput = document.querySelector(".search-text")
-const body = document.querySelector(".body")
+const searchBtn =document.querySelector('.search-button')
+const searchInput = document.querySelector('.search-text')
+const body = document.querySelector('.body')
+const optionsList = document.getElementsByTagName('option')
 
 
-let inputText =""
 const featch =(url ,cb)=>{
-    const xhr = new XMLHttpRequest();
-    xhr.onreadystatechange = () => {
-        if (xhr.readyState === 4) {
-            if (xhr.status === 200) {
-                const data = JSON.parse(xhr.responseText)
-                cb(data)
-            } else {
-                console.log('Error', JSON.parse(xhr.responseText))
-            }
-        }
-    };
+  const xhr = new XMLHttpRequest();
+  xhr.onreadystatechange = () => {
+    if (xhr.readyState === 4) {
+      if (xhr.status === 200) {
+        const data = JSON.parse(xhr.responseText)
+        cb(data)
+      } else {
+        console.log('Error', JSON.parse(xhr.responseText))
+      }
+    }
+  };
 
-    xhr.open('GET', url);
-    xhr.send();
+  xhr.open('GET', url);
+  xhr.send();
 }
 const createCard =(data)=>{
 /// condition to delete the old main if exist
-    if(body.childNodes.length ==18){
-        const bodyElements =body.childNodes
-        bodyElements.item(17).remove()
-    }
+  if(body.childNodes.length ==18){
```

Comment:
what does `18` here refer to? isn't better to assign this value to a variable that has a meaningful name?